### DISCUSSION TITLE
Effects of installing in private repository

### DISCUSSION THREAD

**Discussion by rajtilakjee** [squidfunk/mkdocs-material#5625] [Q&A] [answered]
I have a private repository where I am uploading codes for my latest product. This is not open-source, so I don't want to code available to the general public. However, I would like them to be able to see the documentation, which would contain download links etc.

My question is, can I install Materials for MKDoks in the private repository so that only the documentation is visible to the public? If not, what are the other options available at my disposal?

**Comment by squidfunk** [comment, marked as answer]
As far as I understand your inquiry, yes, you can deploy documentation on GitHub Pages without giving away your sources. Just go through the normal process of deploying a site on GitHub Pages. For other provides, this might be different, though.
